From 84cbbd53e0f8549f9ae27ba37480d72c30e96ad7 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 4 Sep 2018 16:04:14 +0800
Subject: [PATCH 4544/5242] MLK-19442-1 usb: host: xhci: export
 xhci_bus_suspend

commit  2ea59776108835848ef882a8eb77aeba6a520043 from
https://source.codeaurora.org/external/imx/linux-imx.git

It may be used by xhci platform driver, fixed below error when
building xhci as module.

ERROR: "xhci_bus_suspend" [drivers/usb/cdns3/cdns3.ko] undefined!
linux-imx/scripts/Makefile.modpost:92: recipe for target '__modpost' failed
make[2]: *** [__modpost] Error 1
linux-imx/Makefile:1231: recipe for target 'modules' failed
make[1]: *** [modules] Error 2

Reported-by: Bruce Zhang <bo.zhang@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/host/xhci-hub.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/xhci-hub.c b/drivers/usb/host/xhci-hub.c
index 1bbcc9c..6f9b1eb 100644
--- a/drivers/usb/host/xhci-hub.c
+++ b/drivers/usb/host/xhci-hub.c
@@ -1586,6 +1586,7 @@ int xhci_bus_suspend(struct usb_hcd *hcd)
 	spin_unlock_irqrestore(&xhci->lock, flags);
 	return 0;
 }
+EXPORT_SYMBOL(xhci_bus_suspend);
 
 /*
  * Workaround for missing Cold Attach Status (CAS) if device re-plugged in S3.
-- 
1.7.9.5

